manifests/cf-manifest/deployments/900-cf-tests.yml

```diff
@@ -41,6 +41,8 @@ properties:
     include_operator: true
     include_services: true
     include_route_services: false
+    default_timeout: 500
+    nodes: 6
```

Comment:
This was left here by accident. When it comes to effectiveness than ```default_timeout``` definitely helped for some tests but ```nodes: 6``` doesn't seem to have big effect on tests speed as it took 50 minutes to finish.